server/services/operations.go

```diff
@@ -0,0 +1,42 @@
+package services
+
+import "github.com/coinbase/rosetta-sdk-go/types"
+
+const (
+	opGenesisBalanceMovement = "GenesisBalanceMovement"
+	opTransfer               = "Transfer"
+	opFee                    = "Fee"
+	opReward                 = "Reward"
+	opScResult               = "SmartContractResult"
+	opFeeOfInvalidTx         = "FeeOfInvalidTransaction"
+	opFeeRefund              = "FeeRefund"
+)
+
+// SupportedOperationTypes is a list of the supported operations.
+var SupportedOperationTypes = []string{
+	opTransfer, opFee, opReward, opScResult, opFeeOfInvalidTx, opGenesisBalanceMovement, opFeeRefund,
+}
+
+var (
+	// opStatusSuccess is the operation status for successful operations.
+	opStatusSuccess = "Success"
```

Comment:
define all vars in a single `var( ) ` group?

Reply:
Fixed.